components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -51,11 +58,32 @@ func New(signers []ssh.Signer, workspaceInfoProvider p.WorkspaceInfoProvider, he
 
 	server.sshConfig = &ssh.ServerConfig{
 		ServerVersion: "SSH-2.0-GITPOD-GATEWAY",
-		PasswordCallback: func(conn ssh.ConnMetadata, password []byte) (*ssh.Permissions, error) {
+		BannerCallback: func(conn ssh.ConnMetadata) string {
+			if strings.Contains(conn.User(), "#") {
+				return "Welcome use Gitpod. If you see a prompt to enter a password, fill in any value"
```

Comment:
Consider adding new lines after the message, because in Windows it looks a bit messy:

<img width="1539" alt="Screenshot 2022-05-18 at 11 08 57" src="https://user-images.githubusercontent.com/2318450/169003204-82f2115a-a2ea-4f45-acf0-dd106bffc714.png">

Reply:
Do you think it is worth it?

I had users asking already why do you need private key if there is no public. I afraid that next question will be why do you ask password if you don't need it.

Reply:
Agree that this is already cryptic enough.